src/scintillaeditor.cpp

```diff
@@ -540,27 +571,16 @@ void ScintillaEditor::replaceSelectedText(const QString &newText)
     if ((qsci->selectedText() != newText)&&(qsci->hasSelectedText())) qsci->replaceSelectedText(newText);
 }
 
-void ScintillaEditor::replaceAll(const QString &findText, const QString &replaceText)
+void ScintillaEditor::replaceAll(const QString &findText, const QString &replaceText)//QScintilla bug no longer an issue 
```

Comment:
I'm having trouble reproducing this bug in qscinitilla2.9.3. I suppose i could try and add back in a case for "#if QSCINTILLA_VERSION == 0x020903"? my memory is a bit fuzzy on this but i seem to remember selectAll() creating an issue where focus was no longer on the findreplace pannel after the hitting the button, and all my text was still selected. don't know if this is a real issue, but changed it. I suppose i can change it back, or add a seperate compiler condition. let me know what you want